Source/Contrib/ActivityEditor/ActivityEditor/Properties/AssemblyInfo.cs

```diff
@@ -9,8 +9,8 @@
 [assembly: AssemblyDescription("Open Rails Transport Simulator")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Open Rails")]
-[assembly: AssemblyProduct("Open Rails (disabled)")]
```

Comment:
We actually should leave this alone, as this is how this project is hidden from the main menu's dropdown (which is due to it not working properly IIRC).

Reply:
not the most obvious way...

Reply:
You are not wrong there! It actually works by comparing the executable's product name with that of the main menu, and if it is different it is excluded. This was originally to make sure oddities like Visual Studio's "foo.vshost.exe" didn't appear but was co-opted here. Obviously the real solution would be to fix this contributed app or remove it. 🙂